Update repo links (#470)
Shopify/polaris-react: src/components/ExceptionList/README.md

### Opportunity for improvement

> Exception lists aren’t clickable. If you have an idea that could make this component better, reach out to us on our [public repo](https://github.com/shopify/polaris-react/issues).

Reviewer: ```suggestion
Exception lists aren’t clickable. If you have an idea that could make this component better, please [open an issue](https://github.com/shopify/polaris-react/issues).
```
